app/controllers/admin/organizations_controller.rb

```diff
@@ -7,4 +7,58 @@ def index
       per_page: 50
     )
   end
+
+  # show view includes edit
+  def show
+    @org = Organization.find(params[:id])
+    @countries = Country.all
+    @form_action_url = admin_organization_path
```

Comment:
The above two lines are duplicated in many places here. They should be moved into a private method in the controller.